workflows/iuf/operations/vcs-upload-content.yaml

```diff
@@ -0,0 +1,172 @@
+              kubectl apply -n $destination_ns -f -
+              return $?
+          else
+            echo "Didn't find $item_name in the $source_ns namespace"
+            return 1
+          fi
+        }
+        sync_item secret/vcs-user-credentials services vcs-user-credentials-argo argo
+        rc=$?
+        echo $destination_name >> /tmp/secret_name
+        exit $rc
+## gitea-upload-content ##
+  - name: gitea-upload-content
+    nodeSelector:   
+      kubernetes.io/hostname: ncn-m001
+    tolerations:
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    inputs:
+      parameters:
+      - name: vcs_user_credentials_secret_name
+      - name: cf_import_gitea_url
+        value: "https://api-gw-service-nmn.local/vcs"
+      - name: cf_import_product_name
+        value: "{{workflow.parameters.name}}"
+      - name: cf_import_product_version
+        value: "{{workflow.parameters.version}}"
+      - name: cf_import_gitea_org
+        value: cray
+      - name: cf_import_content
+        value: /content
+    container:
+      image: registry.local/artifactory.algol60.net/csm-docker/unstable/cf-gitea-import:1.8.0-alpha.16_a2674cd
+      command:
+      - '/bin/sh'
+      args: ["-c", "/opt/csm/cf-gitea-import/argo_entrypoint.sh"]
+      env:
+      - name: CF_IMPORT_GITEA_USER
+        valueFrom:
+          secretKeyRef:
+            name: "{{inputs.parameters.vcs_user_credentials_secret_name}}"
+            key: vcs_username
+      - name: CF_IMPORT_GITEA_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: "{{inputs.parameters.vcs_user_credentials_secret_name}}"
+            key: vcs_password
+      - name: CF_IMPORT_GITEA_URL
+        value: "{{inputs.parameters.cf_import_gitea_url}}"
+      - name: CF_IMPORT_PRODUCT_NAME
+        value: "{{inputs.parameters.cf_import_product_name}}"
+      - name: CF_IMPORT_PRODUCT_VERSION
+        value: "{{inputs.parameters.cf_import_product_version}}"
+      - name: CF_IMPORT_GITEA_ORG
+        value: "{{inputs.parameters.cf_import_gitea_org}}"
+      - name: CF_IMPORT_CONTENT
+        value: "{{inputs.parameters.cf_import_content}}"
+      - name: CF_IMPORT_FORCE_EXISTING_BRANCH
+        value: "{{inputs.parameters.cf_import_force_existing_branch}}"
+      volumeMounts:
+        - name: content
+          mountPath: /content
+        - name: results
+          mountPath: /results
+        - name: certs # mount cluster certs to ca-certificates.crt for curl/http libraries
+          mountPath: /etc/ssl/certs/ca-certificates.crt
+          subPath: platform-ca-certs.crt
+    volumes:
+    - name: content
+      hostPath: #TODO: Figure out how IUF will pass this path
+        path: "{{=jsonpath(workflow.parameters.content, '$.vcs.path')}}"
+    - name: results #TODO: If admins are allowed to access the results, if not remove this mount
```

Comment:
File a Jira for each TODO, so they are tracked and not lost, please. You can clone the Jira you are working on to get all of the right labels.